File: AFMobile/AFAndroid/app/src/main/java/cz/cvut/fel/matyapav/afandroid/JSONParser.java
package cz.cvut.fel.matyapav.afandroid;

import org.json.JSONObject;

import cz.cvut.fel.matyapav.afandroid.ClassDefinition;

/**
 * Created by Pavel on 17.12.2015.
 */
public interface JSONParser {

    public ClassDefinition parse(JSONObject toBeParsed);
}

File: AFMobile/AFWindowsPhone/AFWindowsPhone/parsers/JSONParser.cs
using AFWindowsPhone.builders.components.parts;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Windows.Data.Json;

namespace AFWindowsPhone.parsers
{
    interface JSONParser
    {
        ClassDefinition parse(JsonObject toBeParsed);
    }
}
